Editor/Analytics/AmplitudeEditorLogger.cs

```diff
@@ -234,13 +233,12 @@ public void LogAvatarCreatorSampleImported()
             LogEvent(EventName.AVATAR_CREATOR_SAMPLE_IMPORTED);
         }
 
-        public void LogPackageInstalled(PackageCoreInfo packageInfo)
+        public void LogPackageInstalled(string id, string name)
```

Comment:
You're not a fan of DTOs are you? 😅 -> https://www.c-sharpcorner.com/article/data-transfer-objects-dtos-in-c-sharp/

Fair enough though, as long as we agree on a consistent approach then I'm happy with whatever convention we use. Do we have a limit on the amount of properties that we pass to a function before we would wrap it in an object?

Reply:
I am which as you see there are way too many structs and class object holding data, but here I didn't wanted Analytics to have a cross dependency on PackageCoreInfo.

Reply:
Also as mentioned before the model namespace caused an issue.

Reply:
I believe this was don't mostly because of issues related to using the PackageCoreInfo object right?
We also weren't using all the props anyways but thats kinda minor

Reply:
Yup, agreed that everything at the moment is using primitives so makes sense to keep it uniform, like you've done 🙂